include/wx/dcsvg.h

```diff
@@ -225,14 +229,14 @@ class WXDLLIMPEXP_CORE wxSVGFileDCImpl : public wxDCImpl
    // their current values in wxDC.
    void DoStartNewGraphics();
 
-   wxFileOutputStream *m_outfile;
+   wxScopedPtr<wxFileOutputStream> m_outfile;
```

Comment:
Then you have to move the initialization of ```m_outfile``` to the constructor initialization list, instead of using the ```Init()``` function. Next question is, why not put all members in the initialization list and remove the ```Init()``` function completely?

Also, ```"wfstream.h"``` will need to be included in ```dcsvg.h``` and forward declaration becomes obsolete. Is this desired?